cob_bringup/components/ros_control_base.launch

```diff
@@ -25,10 +26,12 @@
 		<arg name="reset_errors_before_recover" value="$(arg reset_errors_before_recover)"/>
 	</include>
 
-	<include unless="$(arg sim)" file="$(find cob_omni_drive_controller)/launch/stuck_detector.launch">
-		<arg name="timeout" default="2.0"/>
-		<arg name="threshold" default="0.174533"/>
-	</include>
+	<group unless="$(arg disable_stuck_detector)">
```

Comment:
why do you need to disable the stuck detector for raw3-1? It's an important feature for the FDMs to not be mechanically damaged in case they're stuck. I don't see a reason why that shouldn't work for raw3 FDMs too.